.github/workflows/dev-green.yml

```diff
@@ -1,35 +1,37 @@
-name: Deploy to Dev-green
+name: Deploy to Dev Green
 
 on:
   workflow_dispatch:
+  push:
+    branches: [main]
 
 jobs:
-  Build:
-    runs-on: windows-2019
-    permissions:
-      id-token: write
-      contents: read
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    environment: dev-green-linux
+    concurrency: dev-green-linux
+    env:
+      ECS_CLUSTER: linux-staging
+      ECS_SERVICE: realtime-signs-dev-green
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
-      - uses: mbta/actions/build-push-ecr@v2
+      - uses: actions/checkout@v3
+      - uses: mbta/actions/build-push-ecr@v1
         id: build-push
         with:
-          role-to-assume: ${{ secrets.AWS_ROLE_ARN }}
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
```

Comment:
Realtime Signs doesn't have these secrets stored in the repo environments so I guess we'll have to add them. I thought though that we wanted to avoid saving AWS credentials in GitHub repos directly, or has that changed?

Reply:
You should be using `v2` which uses a different connection type to AWS: https://github.com/mbta/busloc/blob/master/.github/workflows/staging.yml#L25-L34